magma_access_gateway_installer/__init__.py

```diff
@@ -91,3 +99,12 @@ def validate_args(args):
             ip_address(args.gw_ip_address)
         except Exception as e:
             raise e
+
+    if not args.dns:
+        raise ValueError("--dns flag specified, but no addresses given! Exiting...")
+    else:
+        for dns_ip in args.dns:
+            try:
+                ip_address(dns_ip)
+            except Exception as e:
```

Comment:
Not sure why we do the try except if we simply raise, we could simply remove the try/except and only keep `ip_addres(dns_ip)`

Reply:
Agreed. Refactored.